src/lib.rs

```diff
@@ -1182,6 +1182,27 @@ pub unsafe trait Immutable {
 /// }
 /// ```
 ///
+/// # Portability
+///
+/// To ensure consistent endianness for enums with multi-byte representations,
+/// explicitly specify and convert each discriminant:
+///
+/// Here, `DataStoreVersion` enforces little-endian encoding for each `u32`
+/// discriminant, ensuring portability across platforms with differing native endianness.
+///
+/// ```
+/// # use zerocopy_derive::TryFromBytes;
+/// #[derive(TryFromBytes)]
+/// #[repr(u32)]
+/// pub enum DataStoreVersion {
+///     /// Version 1 of the data store.
+///     V1 = 9u32.to_le(),
+///
+///     /// Version 2 of the data store.
+///     V2 = 10u32.to_le(),
+/// }
+/// ```
+///
```

Comment:
Tightening up the prose a bit:
```suggestion
/// # Portability
///
/// To ensure consistent endianness for enums with multi-byte representations,
/// explicitly specify and convert each discriminant using `.to_le()` or
/// `.to_be()`; e.g.:
///
/// ```
/// # use zerocopy_derive::TryFromBytes;
/// // `DataStoreVersion` is encoded in little-endian.
/// #[derive(TryFromBytes)]
/// #[repr(u32)]
/// pub enum DataStoreVersion {
///     /// Version 1 of the data store.
///     V1 = 9u32.to_le(),
///
///     /// Version 2 of the data store.
///     V2 = 10u32.to_le(),
/// }
/// ```
///
```

Reply:
Changes done